projects/gloo/cli/pkg/cmd/options/options.go

```diff
@@ -45,7 +45,9 @@ type Top struct {
 	Zip            bool
 	ErrorsOnly     bool
 	ConfigFilePath string
-	Consul         Consul // use consul as config backend
+	Consul         Consul   // use consul as config backend
+	Selector       string   // label selector for pod scanning
+	Namespaces     []string // namespaces to check
```

Comment:
```suggestion
	PodSelector       string   // label selector for pod scanning
	ResourceNamespaces     []string // namespaces in which to check custom resources
```
nit: clearer naming to differentiate them